lib/digitizer_block_impl.cc

```diff
@@ -700,9 +700,19 @@ namespace gr {
        }
      } catch (const std::exception& ex) {
        d_configure_exception_message = ex.what();
+       GR_LOG_ERROR(d_logger, d_configure_exception_message );
+
+       // It looks like the block is run by gnuradio, no matter if true or false is returned here
```

Comment:
I also couldn't find a single use of `start()` where the result was used (or something other than true was returned) in the gnuradio source.

Reply:
As per the latest [documentation](https://www.gnuradio.org/doc/doxygen-v3.9.4.0/classgr_1_1block.html#a7f58745d1374b30a7b866406dc97850f) the return value of `gr::block.start()` doesn't seem to have a pre-sprecified value and isn't further documented.

I also reckon that the original meaning got lost in the ages. 

Maybe the GnuRadio oracle @marcusmueller can enlighten us ...

Reply:
Resolving the conversation for now ... if we can find out more about it later on, lets open a new MR/issue.